src/background.js

```diff
@@ -173,3 +173,8 @@ browser.storage.local.get()
       // set defaultTheme as initial theme in local storage
       browser.storage.local.set(defaultTheme);
 });
+
+// Handle onClick event for the toolbar button
+browser.browserAction.onClicked.addListener(() => {
+  browser.sidebarAction.open();
```

Comment:
Don't we have a toggle sidebar Action ?

Reply:
I'm not sure what you mean - are you referring to the Sidebar browser action button that looks like this: https://design.firefox.com/icons/viewer/#sidebar?

If so, that toggles the sidebar, but will open whatever page was last opened in the sidebar (so potentially Bookmarks, History, or Notes open). This PR is supposed to add the toolbar button which opens **only** opens Notes. Let me know if that's what you meant or not.